src/Microsoft.AspNet.OData.Shared/PatchMethodHandler.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.OData
+{    
+    /// <summary>
+    /// Base Interface for PatchMethodHandler
+    /// </summary>
+    public interface IPatchMethodHandler
+    {
+
+    }
+
+    /// <summary>
+    /// Handler Class to handle users methods for create, delete and update
```

Comment:
>Handler Class to handle users methods for create, delete and update [](http://example.com/codeflow?start=8&length=67)

add to the description that this is for the clr-backed objects; EdmPatchMethodHandler is used for the non-CLR backed case.